tests/integration/conftest.py

```diff
@@ -281,7 +280,7 @@ def openstack_connection_fixture(clouds_yaml_contents: str) -> Connection:
 @pytest.fixture(scope="module", name="dockerhub_mirror")
 def dockerhub_mirror_fixture(pytestconfig: pytest.Config) -> str:
     """Dockerhub mirror URL."""
-    return pytestconfig.getoption("--dockerhub-mirror", default="")
+    return pytestconfig.getoption("--dockerhub-mirror") or ""
```

Comment:
`default=""` does not work as expected (only considered if the option is not registered)